testsprite_tests/TC013_Inventory_Tracking___Synchronization_with_Shipment_Data.py

```
@@ -0,0 +1,163 @@
+
+        frame = context.pages[-1]
+        # Click the INITIATE SESSION button to login
+        elem = frame.locator('xpath=html/body/div[2]/div[2]/div[3]/form/div[2]/button').nth(0)
+        await page.wait_for_timeout(3000); await elem.click(timeout=5000)
+        
+
+        # -> Try to clear the input fields and re-enter credentials, then click login again to see if it triggers a different response.
+        frame = context.pages[-1]
+        # Clear the Operative ID input field
+        elem = frame.locator('xpath=html/body/div[2]/div[2]/div[3]/form/div/div/div/input').nth(0)
+        await page.wait_for_timeout(3000); await elem.fill('')
```

Ruff (0.14.10)

Multiple statements on one line (semicolon)

(E702)